### ISSUE TITLE
Workflow won't show when use Volta manage Node version

### ISSUE THREAD

**Issue by BrianCodeItUp** [bchatard/alfred-jetbrains#276]
### FAQ

- [X] Yes, my issue is not listed in [troubleshooting guide](https://github.com/bchatard/alfred-jetbrains/blob/main/doc/troubleshooting.md).
- [X] Yes, my issue is not listed in other issues.

### Describe the bug

I use [volta](https://docs.volta.sh/) to manage node version. Workflow doesn't show after install this package as global denpendencies.

### To Reproduce

1. Install node through volta, it should install the lastest lts version of node
```bash
volta install node
```
2. Install `@bchatard/alfred-jetbrains ` as global dependency
```bash
yarn global add  @bchatard/alfred-jetbrains 
```

### Expected behavior

Workflow should show after install `@bchatard/alfred-jetbrains` as global dependency

### Alfred Version

4.6.3

### Node Version

16.5.1

### NPM Version

8.11.0

### Jetbrains Product & Version

WebStrom 2022.1.2

### Installation type

Classic via .dmg

### OS

macOS 12.3.1

**Comment by bchatard** [comment]
hi,

can you try with npm? and also provide the installation trace?

**Comment by BrianCodeItUp** [comment]
Here is the installation trace
<img width="1024" alt="image" src="https://user-images.githubusercontent.com/34977345/173625616-0c76c86b-14bc-44c0-bd4e-01fda6a05aa6.png">

**Comment by bchatard** [comment]
ok, I double check my env & how I install the workflow for the first time & how I update it.

So I have node installed globally with brew, I use it to install / update the workflow (actually node 18.3 / npm 8.12.1).
Then workflow used the version pined in his `package.json`

If I understand the doc of Volta, you should run `volta install @bchatard/alfred-jetbrains` (source: https://blog.volta.sh/2019/06/18/global-installs-done-right/). Before remove completely version install with npm

**Comment by ahoendgen** [comment]
I had the same issue and thought maybe these infos could be helpful others.

I found out that the symlink in the workflows folder is wrong. Volta puts packages in a temp folder which messes things up.

<img width="1199" alt="grafik" src="https://user-images.githubusercontent.com/934583/179708946-ae4f84d7-9f93-4736-8c88-bf46253ef2d8.png">

if you run `npm -g list` you will see that the packages are installed to e.g. `/Users/andre/.volta/tools/image/node/16.14.0/lib`  and not into the `tmp` folder (see screenshot).

Volta seems to have a bug with global installs without a bin in the package.

I solved it this way:

I installed `node` via homebrew but didn't link it so i could keep my volta installation as is.

Then i used the installed npm binary to install the extension e.g. `/opt/homebrew/Cellar/node/18.6.0/libexec/bin/npm install -g @bchatard/alfred-jetbrains`.

Now the workflow should be listed when you run `npm -g list.` Anyway it's not installed in alfred, you still have to go to your `Alfred.alfredpreferences/workflows` folder and symlink the workflow manually e.g. `ln -s /Users/YOUR_USER/.volta/tools/image/node/16.14.0/lib/node_modules/@bchatard/alfred-jetbrains /YOUR/PATH/TO/alfred/Alfred.alfredpreferences/workflows/@bchatard-alfred-jetbrains` 

it took 10 seconds for alfred to recognize the "new" workflow, afterwards it works fine again.
